packages/cli/src/commands/org/list.ts

```diff
@@ -17,7 +17,7 @@ export default class List extends Command {
     ...cli.table.flags(),
   };
 
-  @Preconditions(IsAuthenticated())
+  @Preconditions(HasAccessToken())
```

Comment:
Why this change ? It's not because you have an access token that it is still valid/not expired/will actually be able to list orgs.

With this change, the command will now fail in a cryptic manner for the end user if the token is expired.

Reply:
_Technically_ we're not checking if the user is allowed to list org today.
However, it is true that were dropping on the token validity. Which is where your comment https://github.com/coveo/cli/pull/197#discussion_r625354148 should play its role.

Reply:
Je crois que je vais revert tout ce qui a rapport avec à `IsAuthenticated` et `HasAccessToken `. Pour le changement en soi, j'en ai pas besoin. Je ne change pas le comportement en soi de ça; même si maintenant j'affiche qu'on s'est logged correctement lors du login. Autrefois, sans org, ça affichait un message de fail, mais après tu pouvais faire des commandes ne nécéssitant pas de specific org.

Je suggèrerais qu'on fasse la validation des orgs pour les commandes qui en ont besoin dans un autre changement. Qu'en dites-vous?

Reply:
I agree with the idea that the change between IsAuthenticated and HasToken should be dealt with in another PR.
However I would consider this change to be a prerequisite for this PR to go forward.